docs/usage/delete.rst.inc

```diff
@@ -68,6 +68,7 @@ borg delete
 
     optional arguments
         -n, --dry-run    do not change repository
+        --list          list the archives/files which would be deleted
```

Comment:
read line 1 of this file - no need to commit this and make the PR review more work, all these files will be auto-generated before a release.